packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -123,7 +123,8 @@ export async function fetchHelper(
     }, (error) => {
         const online = isOnline();
         const errorText = `${error}`;
-
+        const spoRegex = /((http|https):\/\/.*\.com)/i;
```

Comment:
```suggestion
        const urlRegex = /((http|https):\/\/.*\.com)/i;
```

Reply:
And if you can not restrict it to end in .com that would be awesome.  Maybe this - instead of `.*` do an explicit character set of all valid URL characters (look it up, don't try to figure it out!).  Like if the set of chars after https:// was just letters you would do `[a-z]+` instead of `.*`  Then if there's whitespace or end of line or whatever, it will stop matching at that point.

Reply:
@jatgarg drawing your attention to this ^^ about not depending on ".com"

Reply:
Ok. I will change the regex depending upon this suggestion.

Reply:
I think the answer is no, but for the sake of completeness, do we need to worry about any other protocols? Since we know what the thing on the other side is (ODSP) we probably know the kind of requests that might happen and thus what we might see in the logs, so ftp/sftp/file feel (probably) unnecessary. Maybe ws?